configure.ac

```diff
@@ -36,6 +36,9 @@ AC_CONFIG_HEADERS([src/lib/config.h])
 
 # test for TSS dependecies
 PKG_CHECK_MODULES([TSS2_FAPI],   [tss2-fapi])
+PKG_CHECK_MODULES([TSS2_FAPI_3_0],   [tss2-fapi >= 3.0],
+                  [AC_DEFINE([FAPI_3_0], [1], [Fapi3.0])],
+                  [AC_MSG_WARN([FAPI 3.0 features not available.])])
```

Comment:
This just ends up as a backend error anyways, so make configure unhappy about it

Reply:
Well, this is a warning. I guess I could make this warning more elaborate.
Or should I just set it to no-problem ?